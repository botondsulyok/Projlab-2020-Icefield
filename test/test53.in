field n1 0
field n2 0
neighbours n1 n2
neighbours n2 n1
field spawn 0
entity esk Eskimo spawn
entity pb icebear n1
stats n1
stats n2
stats pb
stats spawn
stats esk
current
turn
stats n1
stats n2
stats pb
stats spawn
stats esk
current