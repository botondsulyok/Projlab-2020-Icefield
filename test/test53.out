id: n1
type: iceunit
building: plains0
snow: 0
pickable: null
standers: pb
pullers: null
masters: null
neighbours: n2

id: n2
type: iceunit
building: plains1
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: n1

id: pb
type: icebear
iceUnit: n1
movebehaviour: amph0

id: spawn
type: iceunit
building: plains2
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

esk

id: n1
type: iceunit
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: n2

id: n2
type: iceunit
building: plains1
snow: 0
pickable: null
standers: pb
pullers: null
masters: null
neighbours: n1

id: pb
type: icebear
iceUnit: n2
movebehaviour: amph0

id: spawn
type: iceunit
building: plains2
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

esk

